source/mainnet/net/resources/glossary.rst

```diff
@@ -275,16 +282,17 @@ An invocable function of the smart contract that usually takes arguments. Each e
 Epoch
 =====
 
-A time period consisting of multiple :ref:`slots<glossary-slot>`. An epoch is one hour on testnet and mainnet. At the start of each epoch, a :ref:`leadership election nonce<glossary-leader-election>` is computed based on the block nonces of the previous epoch.
+A time period consisting of multiple :ref:`rounds<glossary-round>`. An epoch is one hour on testnet and mainnet. At the start of each epoch, a :ref:`leadership election nonce<glossary-leader-election>` is computed based on the block nonces of the previous epoch.
```

Comment:
As I wrote above, a round is not a unit of time. I'd remove that here and just say an epoch is a time period, 1 hour on testnet etc.